src/Components/Notification.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+
+interface NotificationPropsType {
+  hasErrorFromServer: boolean,
+  clearNotification: () => void,
+  errorMessage: string,
+}
+
+export const Notification: React.FC<NotificationPropsType> = ({
+  hasErrorFromServer,
+  clearNotification,
+  errorMessage,
+}) => {
+  const [hidden, setHidden] = useState(true);
+
+  useEffect(() => {
+    if (hasErrorFromServer) {
+      setHidden(false);
+      const timerId = window.setTimeout(() => {
```

Comment:
You should clear the timeout on the component unmount to avoid unpredictable side effects